app/views/mail/jobSuccessfulUploadConvert.scala.html

```diff
@@ -31,6 +31,17 @@
 <p>
   Did you know that we offer services for <a href="https://webknossos.org/services/alignment">dataset alignment & stitching</a> and <a href="https://webknossos.org/services/automated-segmentation">AI image segmentation</a> to kickstart your research?
 </p>
+<div style="text-align: center; margin-top: 10px; margin-bottom: 6px;">
```

Comment:
unrelated to this pr, but in line 9 of this file, it says `amongst your other dataset` but it should be datasetS.

Reply:
Fixed